src/condor_scripts/adstash/elastic.py

```diff
@@ -212,12 +194,23 @@ def __init__(
                 logging.error("Using HTTPS with Elasticsearch requires that either ca_certs be provided or certifi library be installed")
                 sys.exit(1)
             es_client["use_ssl"] = True
-            es_client["verify_certs"] = True
+            es_client["verify_certs"] = False
 
-        self.handle = elasticsearch.Elasticsearch([es_client])
-        super().__init__(self, logdir)
+        if use_opensearch:
+            self.use_opensearch = True
```

Comment:
I had to give the ElasticInterface class a new attribute in order to use the use_opensearch argument in make_mapping() on line 207